README.asciidoc

```diff
@@ -110,3 +110,7 @@ or a <<init-script,Gradle>> instance.
 A `settings.gradle` script. A https://docs.gradle.org/current/javadoc/org/gradle/api/initialization/Settings.html[settings script]
 has a large number of responsibilities, but one of the most important is declaring the set of <<project,projects>>
 that are part of the build, via `include ':project1'` and so on.
+
+[[tasks]]
+=== Tasks (a task)
+Each <<project,projects>> is made up of one or more tasks. Each task is atomic, with inputs and outputs. Gradle executes tasks to perform its work. Task examples include; compiling classes, creating a JAR/APK, generating Javadoc, running static analysis (lint), deleting temporary files, or publishing to a repository, etc. When a task is asked to run we can see the outcome of the task. This will be one of EXECUTED, SKIPPED, FAILED, FROM-CACHE, UP-TO-DATE, NO-SOURCE or _blank_ (meaning executed). 
```

Comment:
Some suggestions:
* "Each task is atomic" -> "Each task ought to be atomic (but often isn't)"
* "Task examples include;" -> "Task examples include:" (from `;` to `:`)
* "compiling..." -> "compiling source code, creating artifacts (such as jars or apks), generating Javadoc, running static analysis (e.g., lint)"

Note that I'm not sure about JAR vs jar and APK vs apk. My parenthetical `(but often isn't)` is meant to allude to typical user-error in writing tasks, and the fact that Gradle doesn't prevent such error.

Reply:
- done
- done
- done
👍